wtpython/display.py

```diff
@@ -122,28 +126,30 @@ def create_body_text(self) -> RenderableType:
         output = Markdown(text, inline_code_lexer="python")
         return output
 
+    async def update_body(self) -> None:
+        """Update the body"""
+        await self.body.update(self.create_body_text())
+        self.body.y = 0
+        self.body.target_y = 0
+
     async def action_next_question(self) -> None:
         """Go to the next question"""
-        if len(self.data["results"]) > self.index + 1:
+        if len(SO_RESULTS) > self.index + 1 and self.viewing_traceback:
```

Comment:
Why do we need to check if the traceback is being viewed here? To me, either
(1) we always change the question, in which case this is not needed
(2) we only change the question when they're actually looking at the question, in which case the condition should be `not self.viewing_traceback`?

Reply:
It now always switches the questions, and if they are viewing the traceback, it views the question instead of the traceback.